raiden-ts/tests/e2e/raiden.spec.ts

```diff
@@ -762,13 +762,6 @@ describe('Raiden', () => {
       ).rejects.toThrowError(/Invalid path parameter./i);
     });
 
-    test('target not available', async () => {
-      expect.assertions(1);
-      await expect(raiden.transfer(token, partner, 21)).rejects.toThrowError(
```

Comment:
Hmm, do we wanna remove this test? Isn't it expected to reject if the target is offline?

Reply:
I did already in the recent fixup commit. :+1: 
I also think it is of no value anymore and must not be replaced.